test/input_gen/transLayer_v2.py

```diff
@@ -50,8 +50,8 @@ def transpose_(weight):
         new_params = [transpose_(params[0])]
     yield from new_params
 
-@register_for_(torch.nn.BatchNorm1d)
-def bn1d_translate(model):
+@register_for_((torch.nn.BatchNorm1d, torch.nn.BatchNorm2d))
+def bn_translate(model):
```

Comment:
Is n't it same file with pyutils/transLayer_v2.py? If it is then it might be better to place it into tools with proper naming. 
How about TorchConverter?

Reply:
> Is n't it same file with pyutils/transLayer_v2.py? If it is then it might be better to place it into tools with proper naming. How about TorchConverter?

Okay, then I will move this script to tools/pyutils/TorchConverter.py for using common module. thanks!

Reply:
It's done. thank you :)